explainaboard/loaders/named_entity_recognition.py

```diff
@@ -14,5 +19,25 @@ class NERLoader(Loader):
         please refer to `test_loaders.py`
     """
 
-    _default_file_type = FileType.conll
-    _default_file_loaders = {FileType.conll: CoNLLFileLoader()}
+    @classmethod
+    def default_dataset_file_type(cls) -> FileType:
+        return FileType.conll
+
+    @classmethod
+    def default_output_file_type(cls) -> FileType:
+        return FileType.conll
+
+    @classmethod
+    def default_dataset_file_loaders(cls) -> dict[FileType, FileLoader]:
+        return {
+            FileType.conll: CoNLLFileLoader(
+                [
+                    FileLoaderField(0, "tokens", str),
+                    FileLoaderField(1, "true_tags", str),
+                ]
+            )
+        }
+
+    @classmethod
+    def default_output_file_loaders(cls) -> dict[FileType, FileLoader]:
+        return {FileType.conll: CoNLLFileLoader([FileLoaderField(0, "pred_tags", str)])}
```

Comment:
For CoNLL format it'd be a bit weird to just output the tags (very non-standard), so I think we should probably assume that it'll still have the tokens in the first column.

Reply:
Got it! I added the token column back to the output file for CoNLL.